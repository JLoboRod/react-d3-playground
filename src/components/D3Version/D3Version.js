import React from 'react';
import * as d3 from 'd3';

const D3Version = () => <span>D3 v{d3.version}</span>;

export default D3Version;
