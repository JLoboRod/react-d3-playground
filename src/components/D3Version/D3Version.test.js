import React from 'react';
import { shallow } from 'enzyme';
import D3Version from './D3Version';

it('renders without crashing', () => {
  const wrapper = shallow(<D3Version />);

  expect(wrapper).toMatchSnapshot();
});
