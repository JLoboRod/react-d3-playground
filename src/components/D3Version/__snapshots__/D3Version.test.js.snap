// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<span>
  D3 v
  5.9.2
</span>
`;
